HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.3.3 (2020-08-12)
```

Comment:
normally, we're putting her `not released yet`, instead of date, but it does not matter, we'll release the new version immediatelly.